pkg/devicemapper/devmapper.go

```diff
@@ -264,14 +259,6 @@ func UdevWait(cookie *uint) error {
 	return nil
 }
 
-var dmLogger DevmapperLogger
-
-// LogInit initializes the logger for the device mapper library.
-func LogInit(logger DevmapperLogger) {
-	dmLogger = logger
-	LogWithErrnoInit()
-}
```

Comment:
I like the idea of retaining `LogInit()` but changing the semantics so that caller can register its own logger and get all the messages. IOW, it replaced logrus. 

So if existing users were using it without any logger `LogInit()`, they will not see any change except that logs will start going to journal. Those who were registering their own logger, they will also not see any change as they will continue to get all the logs and then they can decide what to do with those logs.

IOW, default logger of pkg/devicemapper will be `logrus` until and unless overwridden by `LogInit`. 

Does this sound reasonable to you?